### PR TITLE
Add ESLint rule for throwing only error objects

### PR DESCRIPTION
**Pull request by rluvaton**
How do you propose that I should state that in case the readers use TS they should use the `@typescript-eslint/no-throw-literal` as it solve the regular rule limitation.

https://github.com/typescript-eslint/typescript-eslint/issues/1258

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -127,6 +127,8 @@ Read in a different language: [![CN](/assets/flags/CN.png)**CN**](/README.chines
 
 **TL;DR:** Many throw errors as a string or as some custom type – this complicates the error handling logic and the interoperability between modules. Whether you reject a promise, throw an exception or emit an error – using only the built-in Error object (or an object that extends the built-in Error object) will increase uniformity and prevent loss of information
 
+There is `no-throw-literal` ESLint rule that strict that (although it have some [limitations](https://eslint.org/docs/rules/no-throw-literal))
```
Comment (kevynb) [inline README.md:130]: I think we're missing a `check` 🙂 : `that strictly checks that`

Comment (kevynb) [reply README.md:130]: Also, let's add you sentence right after `loss of information.` We tend to avoid line breaks on the main page

**Review by kevynb — COMMENTED** [review]
That's a nice addition 👍  Mentioning the typescript rule too could be good.

File: README.md [hunk 2]
```diff
@@ -125,9 +125,7 @@ Read in a different language: [![CN](/assets/flags/CN.png)**CN**](/README.chines
 
 ## ![✔] 2.2 Use only the built-in Error object
 
-**TL;DR:** Many throw errors as a string or as some custom type – this complicates the error handling logic and the interoperability between modules. Whether you reject a promise, throw an exception or emit an error – using only the built-in Error object (or an object that extends the built-in Error object) will increase uniformity and prevent loss of information
-
-There is `no-throw-literal` ESLint rule that strict that (although it have some [limitations](https://eslint.org/docs/rules/no-throw-literal))
+**TL;DR:** Many throw errors as a string or as some custom type – this complicates the error handling logic and the interoperability between modules. Whether you reject a promise, throw an exception or emit an error – using only the built-in Error object (or an object that extends the built-in Error object) will increase uniformity and prevent loss of information. There is `no-throw-literal` ESLint rule that strictly checks that (although it have some [limitations](https://eslint.org/docs/rules/no-throw-literal) which can be solved when using TypeScript and setting the `@typescript-eslint/no-throw-literal` rule)
```
Comment (elektronik2k5) [inline README.md:128]:
```suggestion
**TL;DR:** Many throw errors as a string or as some custom type – this complicates the error handling logic and the interoperability between modules. Whether you reject a promise, throw an exception or emit an error – using only the built-in Error object (or an object that extends the built-in Error object) will increase uniformity and prevent loss of information.

There is a `no-throw-literal` ESLint rule that strictly checks that (although it has some [limitations](https://eslint.org/docs/rules/no-throw-literal), which can be solved when using TypeScript and using the `@typescript-eslint/no-throw-literal` rule).
```

**Review by elektronik2k5 — DISMISSED** [review]

File: README.md (hunk 2 above)
Comment (kevynb) [reply README.md:128]: Thank you @elektronik2k5 for your suggestion however we try to keep a two block per bullet point "style". One `tl;dr` and one `otherwise`. Sometimes we also add examples to those two but that's it, no extra linebreaks.

**Review by kevynb — APPROVED** [review]
Good addition @rluvaton, thanks for submitting the PR! Let's merge 🔥
